src/Appwrite/Utopia/Response/Model/Message.php

```diff
@@ -17,12 +17,26 @@ public function __construct()
                 'default' => '',
                 'example' => '5e5ea5c16897e',
             ])
-            ->addRule('to', [
+            ->addRule('topics', [
```

Comment:
btw, $createdAt and $updatedAt probably need to be included in these new models too.

Reply:
added this